src/compiler/output-targets/test/output-targets-dist.spec.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
this is debatable I think. I added `// @ts-nocheck` to the files in `src/compiler/output-targets/test` because they are all skipped currently and getting them passing again is a substantial amount of work (which we should do!).

I think at the same time that we get these passing we can remove this and get the typescript compiler passing here too.

Reply:
There are a few other tests in other directories to which I did this too, but they are also files with `xdescribe`

Reply:
yup agreed, I took a quick stab at getting one of these going (I think it was this file here, `output-target-dist.spec.ts`) and it isn't exactly trivial, especially since we don't really have good tests written right now, so it's not just a matter of getting what is there to pass.